pkg/gatherers/clusterconfig/machine_sets.go

```diff
@@ -52,9 +55,27 @@ func gatherMachineSet(ctx context.Context, dynamicClient dynamic.Interface) ([]r
 		}
 		records = append(records, record.Record{
 			Name: recordName,
-			Item: record.ResourceMarshaller{Resource: &machineSets.Items[i]},
+			Item: record.ResourceMarshaller{Resource: anonymizeMachineset(&machineSets.Items[i])},
 		})
 	}
 
 	return records, nil
 }
+
+func anonymizeMachineset(data *unstructured.Unstructured) *unstructured.Unstructured {
+	fieldsToAnonymize := [][]string{
+		{"spec", "template", "spec", "providerSpec", "value", "projectID"},
+		{"spec", "template", "spec", "providerSpec", "value", "region"},
```

Comment:
I was experimenting with adding one more line:
`{"spec", "template", "spec", "providerSpec", "value", "serviceAccounts"},`
but this will render empty `serviceAccounts` (because there's slice of strings not a string). It would be nice to anonymize the serviceAccounts as well IMO, because there's an email address.

Reply:
done